pennylane_lightning/core/src/simulators/lightning_tensor/tensor/base/TensorBase.hpp

```diff
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <functional>
```

Comment:
Why was functional included here? Is it missing from a previous definition?

Reply:
It's required for the `std::multiplies<>` [functor](https://github.com/PennyLaneAI/pennylane-lightning/blob/99ec44ed6d01dc9bf5c32b414cdf3a7f758a1a93/pennylane_lightning/core/src/simulators/lightning_tensor/tensor/base/TensorBase.hpp#L49) here. It was missing from a previous definition.